Code/DataStructs/Wrap/testDiscreteValueVect.py

```diff
@@ -251,8 +251,8 @@ def test9ToNumpy(self):
     bv[31] = 12
     arr = numpy.zeros((3, ), 'i')
     ds.ConvertToNumpyArray(bv, arr)
-    for i in range(len(bv)):
-      self.assertEqual(bv[i], arr[i])
+    for bbv, aarr in zip(bv, arr):
```

Comment:
Same comment here. I think this can be a single assertEqual

Reply:
apologies, it looks like that didn't work as simply as I hoped it would. I apologize for the unhelpful suggestion.